CMakeLists.txt

```diff
@@ -326,29 +326,50 @@ IF(NOT CYCLUS_DOC_ONLY)
     ####################################### begin cpack ##########################################
     ##############################################################################################
 
-    SET(CPACK_COMPONENTS_ALL cyclus testing libraries data core)
 
     # Here's where we package it with CPack
     SET(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Cyclus : A nuclear fuel cycle simulator from UW-Madison.")
-
+    
+    # Here we set some components for installation with cpack
+    SET(CPACK_COMPONENTS_ALL cyclus testing libraries data core)
     SET(CPACK_GENERATOR "DEB")
+
+    # Organisation
     SET(CPACK_DEBIAN_PACKAGE_MAINTAINER "CNERG, UW-Madison") #required
     SET(CPACK_PACKAGE_VENDOR "CNERG, UW-Madison")
     SET(CPACK_PACKAGE_DESCRIPTION_FILE "${PROJECT_SOURCE_DIR}/README.rst")
     SET(CPACK_RESOURCE_FILE_LICENSE "${PROJECT_SOURCE_DIR}/LICENSE.rst")
+
+    # Version
     SET(CPACK_PACKAGE_VERSION_MAJOR "1")
-    SET(CPACK_PACKAGE_VERSION_MINOR "0")
-    #SET(CPACK_PACKAGE_VERSION_PATCH "0") # should use commit number here  ?
-    SET(CPACK_PACKAGE_NAME "${CMAKE_PROJECT_NAME}")
+    SET(CPACK_PACKAGE_VERSION_MINOR "4")
+    SET(CPACK_PACKAGE_VERSION_PATCH "0") 
+
+    # Dependencies
+    SET(CPACK_DEBIAN_PACKAGE_DEPENDS "libtcmalloc-minimal4 (>= 2.1), libboost-filesystem-dev (>= 1.54.0), libboost-program-options-dev (>= 1.54.0), libboost-serialization-dev (>= 1.54.0), libhdf5-dev (>= 1.8.11), libxml++2.6-dev (>= 2.36.0), coinor-libcbc-dev (>= 2.8.7)")
+
+
+    SET(CPACK_PACKAGE_NAME "cyclus")
+    SET(
+      CPACK_PACKAGE_INSTALL_DIRECTORY
+      "cyclus_${CPACK_PACKAGE_VERSION_MAJOR}.${CPACK_PACKAGE_VERSION_MINOR}.${CPACK_PACKAGE_VERSION_PATCH}"
+      )
+    
+    SET(
+      CPACK_PACKAGE_FILE_NAME
+      "${CPACK_PACKAGE_NAME}_${CPACK_PACKAGE_VERSION_MAJOR}.${CPACK_PACKAGE_VERSION_MINOR}.${CPACK_PACKAGE_VERSION_PATCH}_${CPACK_HASH_NAME}_${CPACK_SYSTEM_NAME}"
```

Comment:
Could this be reworked as 2 or 3 set commands so that each fits on a line?  Is it worth it?